src/index.html

```diff
@@ -6,9 +6,61 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo_link">
+        <img src="./images/logo.png" alt="MOYO LOGO">
+      </a>
+
+      <nav class="navigation">
+        <ul class="list">
+          <li class="elements">
+            <a href="#" class="link is-active">apple</a>
+          </li>
+
+          <li class="elements">
+            <a href="#" class="link">samsung</a>
+          </li>
+
+          <li class="elements">
+            <a href="#" class="link">smartphones</a>
+          </li>
+
+          <li class="elements">
+            <a
+            href ="#"
+            class="link"
+            data-qa="hover"
+            >
+            laptops & computers
+            </a>
```

Comment:
Check [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)
![image](https://github.com/mate-academy/layout_moyo-header/assets/91411692/70125243-32e2-42d6-86f8-72a467b768df)

```suggestion
            <a
              href ="#"
              class="link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```